docs/Reference/Pantheon-API-Methods.md

```diff
@@ -739,7 +781,21 @@ Returns the account balance of the specified address.
       "result" : "0x0"
     }
     ```
-
+    
+    ```bash tab="curl GraphQL request"
+    curl -X POST -H "Content-Type: application/json" --data '{ "query": "{ account ( address: \"0xfe3b557e8fb62b89f4916b721be55ceb828dbd73\") { balance } }"}' http://localhost:8547/graphql
+    ```
+    
+    ```bash tab="GraphQL result"
```

Comment:
I think the fix will be enough.  Heaps of these examples will result in errors because they assume specific accounts/blocks/transactions exist.  I think adding specific notes on this every time will clutter the doc for not a huge amount of gain.  Happy to discuss if you think we should add them though.